base/essentials.jl

```diff
@@ -148,6 +148,24 @@ macro boundscheck(blk)
       $(Expr(:boundscheck,:pop)))
 end
 
+"""
+    @inbounds(blk)
+
+In the example below the bound check of array A is skipped to improve performance.
+```julia
+    function sum(A::AbstractArray)
+    r = zero(eltype(A))
+    for i = 1:length(A)
+    @inbounds r += A[i]
+    end
+    return r
+    end
```

Comment:
No problem :). I've made the changes. Shall I still start an issue about the indent in code blocks?